crates/uv-cli/src/lib.rs

```diff
@@ -2711,23 +2712,27 @@ pub struct RunArgs {
     #[arg(long, conflicts_with_all = ["only_group", "only_dev"])]
     pub group: Vec<GroupName>,
 
-    /// Exclude dependencies from the specified dependency group.
+    /// Disable the specified dependency group.
+    ///
+    /// This is specifically for disabling individual default groups or to
+    /// modify `--all-groups`. It also has priority over `--group`.
     ///
     /// May be provided multiple times.
     #[arg(long)]
     pub no_group: Vec<GroupName>,
 
-    /// Exclude dependencies from default groups.
+    /// Don't implicitly include the default dependency groups.
     ///
-    /// `--group` can be used to include specific groups.
+    /// `--group` can still be used to add back individual defaulted groups,
+    /// making this unlike `--no-group <some-default-group>`.
```

Comment:
Then, following above, we can say....

> uv includes the groups defined in `tool.uv.default-groups` by default. This disables that option, however,
> specific groups can still be included with `--group`.

We could have something like

> In contrast, `--no-group` cannot be overridden by `--group`.

but I think it's kind of too in the weeds here? It makes more sense to cover it in `--group` / `--no-group`.

Reply:
Hmm I agree on removing the second paragraph as too in-the-weeds.

Reply:
Or rather I'm taking what you said here in the first quote.